src/globus_sdk/services/search/data.py

```diff
@@ -115,6 +115,9 @@ def __init__(
         additional_fields: dict[str, t.Any] | None = None,
     ):
         super().__init__()
+        exc.warn_deprecated(
+            "'SearchQuery' is a deprecated name. Use 'SearchQueryV1' instead."
```

Comment:
```suggestion
            "'SearchQuery' is deprecated. Use 'SearchQueryV1' instead."
```

This is a minor edit to the warning string, but semantically significant. `SearchQuery` and `SearchQueryV1` are not different names for the same thing. They are different things.